src/rpc/net.cpp

```diff
@@ -242,13 +242,13 @@ static UniValue disconnectnode(const JSONRPCRequest& request)
     if (request.fHelp || request.params.size() == 0 || request.params.size() >= 3)
         throw std::runtime_error(
             "disconnectnode \"[address]\" [nodeid]\n"
-            "\nImmediately disconnects from the specified peer node.\n"
-            "\nStrictly one out of 'address' and 'nodeid' can be provided to identify the node.\n"
-            "\nTo disconnect by nodeid, either set 'address' to the empty string, or call using the named 'nodeid' argument only.\n"
-            "\nArguments:\n"
+            "\n" "Immediately disconnects from the specified peer node.\n"
+            "\n" "Strictly one out of 'address' and 'nodeid' can be provided to identify the node.\n"
+            "\n" "To disconnect by nodeid, either set 'address' to the empty string, or call using the named 'nodeid' argument only.\n"
+            "\n" "Arguments:\n"
```

Comment:
Why are these changes necessary? There are plenty more leading `\n` in this file.

Reply:
The old Ubuntu packaged version didn't understand `\n` and thus flagged "nTo" as a typo. That seems to have been fixed in the more recent `pip` installed version.